### PR TITLE
Added sms api for signal-cli

### PR DESCRIPTION
**Pull request by mfulz**
Simple sms api for using signal-cli https://github.com/AsamK/signal-cli

### CODE REVIEW

**Comment by coudot** [comment]
Hi @mfulz 

thanks for this contribution!

Could you add some documentation? See https://github.com/ltb-project/self-service-password/blob/master/docs/sms_api.rst

**Comment by mfulz** [comment]
> Hi @mfulz
> 
> thanks for this contribution!
> 
> Could you add some documentation? See https://github.com/ltb-project/self-service-password/blob/master/docs/sms_api.rst

Done :)

File: lib/smsapi-signal-cli.inc.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ * @return 1 if message sent, 0 if not
+ */
+function send_sms_by_api($mobile, $message) {
+    global $signal_user, $signal_config, $signal_cli;
+    if (!$signal_user || !$signal_config || !$signal_cli) {
+      error_log('Trying to access signal without credentials. Set signal_user, signal_config and signal_cli in your config.inc.local.php');
+      return 0;
+    }
+
+    $cmd = $signal_cli . " -u \"" . $signal_user . "\" send -m " . "\"" . $message . "\" \"" . $mobile . "\"";
```
[the remark this answers is not in the document]
Comment (mfulz) [reply lib/smsapi-signal-cli.inc.php]: Good point done

**Comment by coudot** [comment]
Great, thanks a lot!

**Pull request merged**